jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/definition/list/ProcessDefinitionListViewImpl.java

```diff
@@ -166,16 +166,16 @@ public String getValue(ProcessSummary object) {
         return versionColumn;
     }
 
-    private Column initProjectColumn() {
-        Column<ProcessSummary, String> projectColumn = new Column<ProcessSummary, String>(new TextCell()) {
+    private Column initDeploymentColumn() {
+        Column<ProcessSummary, String> deploymentColumn = new Column<ProcessSummary, String>(new TextCell()) {
             @Override
             public String getValue(ProcessSummary object) {
                 return object.getDeploymentId();
             }
         };
-        projectColumn.setSortable(true);
-        projectColumn.setDataStoreName(COL_ID_PROJECT);
-        return projectColumn;
+        deploymentColumn.setSortable(true);
+        deploymentColumn.setDataStoreName(COL_ID_DEPLOYMENT);
```

Comment:
@bxf12315 if you keep the COL_ID_PROJECT in here, you could avoid the backend change, right?